oneandone/src/main/java/org/apache/jclouds/oneandone/rest/OneAndOneProviderMetadata.java

```diff
@@ -66,6 +69,7 @@ public static Properties defaultProperties() {
       properties.put(POLL_MAX_PERIOD, 2L * 10L);
       properties.put(PROPERTY_SO_TIMEOUT, 60000 * 5);
       properties.put(PROPERTY_CONNECTION_TIMEOUT, 60000 * 5);
+      properties.put(PROPERTY_MAX_RATE_LIMIT_WAIT, 20);
```

Comment:
Note that the value is in milliseconds, so under a rate limited environment this is likely to have no effect.